### PR TITLE
"Display annotation" sample

### PR DESCRIPTION
**Pull request by PaulAllanSturm**
The new "Display annotation" sample

### CODE REVIEW

File: assets/generated_samples_list.json [hunk 1]
```diff
@@ -992,7 +1021,7 @@
       "OGC",
       "WMS",
       "web map service",
-      "ArcGISMapViewController.identifyLayer",
+      "ArcGISMapViewController.identifyLayerAsync",
```
Comment (kmueller-gis) [inline assets/generated_samples_list.json:1024]: Should not have `Async`

File: lib/samples/display_annotation/display_annotation.dart [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:flutter/material.dart';
+
+void main() async {
```
Comment (kmueller-gis) [inline lib/samples/display_annotation/display_annotation.dart:19]: `main` function should be removed

File: lib/samples/display_annotation/display_annotation.dart [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  // Supply your apiKey using the --dart-define-from-file command line argument
+  ArcGISEnvironment.apiKey = const String.fromEnvironment('API_KEY');
+
+  runApp(
```
Comment (kmueller-gis) [inline lib/samples/display_annotation/display_annotation.dart:23]: `runApp` should be removed

File: assets/generated_samples_list.json (hunk 1 above)
Comment (jenmerritt) [reply assets/generated_samples_list.json:1024]: That was a lingerer from a sample I did and missed. You could merge `v.next` and re-run the script as I've merged that now.

File: lib/samples/display_annotation/README.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+## How to use the sample
+
+Pan and zoom to see names of waters and burns in a small region of Scotland.
+
+## How it works
+
+1. Create an `ArcGISMap` with a light gray canvas and a viewpoint near the data.
+2. Create an `AnnotationLayer` from a feature service URL.
+3. Add both layers to the operational layers of the map and add it to a `MapView`.
```
Comment (jenmerritt) [inline lib/samples/display_annotation/README.md:19]:
```suggestion
3. Add both layers to the operational layers of the map and add it to an `ArcGISMapView`.
```

File: lib/samples/display_annotation/display_annotation.dart [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+class DisplayAnnotation extends StatefulWidget {
+  const DisplayAnnotation({super.key});
+
+  @override
+  State<DisplayAnnotation> createState() => _DisplayAnnotationState();
+}
+
+class _DisplayAnnotationState extends State<DisplayAnnotation> {
```
Comment (jenmerritt) [inline lib/samples/display_annotation/display_annotation.dart:37]:
```suggestion
class _DisplayAnnotationState extends State<DisplayAnnotation> with SampleStateSupport {
```
Hopefully sprint x's and more hands on deck we can get more scripted and/or github checks for us in samples.

File: lib/samples/display_annotation/display_annotation.dart (hunk 2 above)
Comment (PaulAllanSturm) [reply lib/samples/display_annotation/display_annotation.dart]: gone

File: lib/samples/display_annotation/display_annotation.dart (hunk 3 above)
Comment (PaulAllanSturm) [reply lib/samples/display_annotation/display_annotation.dart]: gone

File: assets/generated_samples_list.json (hunk 1 above)
Comment (PaulAllanSturm) [reply assets/generated_samples_list.json]: script re-runned

**Review by kmueller-gis — APPROVED** [review]

**Review by jenmerritt — APPROVED** [review]

**Pull request merged**